src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+@import 'blocks/header';
+@import 'blocks/mixins-variables-fonts';
```

Comment:
Consider importing `mixins-variables-fonts` before any other styles that might use these variables or mixins. This ensures that all variables and mixins are defined before they are used in other styles.